### PR TITLE
Migrates Skeleton component from AntD

### PR DESCRIPTION
**Pull request by jvorcak**
This PR Migrates `Skeleton` to Chakra's `SkeletonText`. I think the look'n'feel is very similar to the original.

Screenshot 2023-10-27 at 9 19 35
Screenshot 2023-10-27 at 9 19 16

### CODE REVIEW

File: frontend/src/components/pages/overview/Overview.tsx [hunk 1]
```diff
@@ -213,20 +212,23 @@ class Overview extends PageComponent {
                             </ul>
 
                             <ul className="resource-list">
-                                {news
-                                    ? news.map((x, i) => <li key={i}>
-                                        <a href={x.url} rel="noopener noreferrer" target="_blank" className="resource-link">
+                                <SkeletonText
+                                    isLoaded={Boolean(news)}
+                                    noOfLines={4}
```
Comment (malinskibeniamin) [inline frontend/src/components/pages/overview/Overview.tsx:217]: I see we had 3 lines previously, not sure if it's a big deal.

File: frontend/src/components/pages/overview/Overview.tsx [hunk 2]
```diff
@@ -261,8 +263,10 @@ const ResourcesBadge = (p: { type?: string | undefined }) => {
 function ClusterDetails() {
     const overview = api.clusterOverview;
     const brokers = api.brokers;
-    if (!overview || !brokers)
-        return <Skeleton paragraph={{ rows: 16 }} />
+
+    if (!overview || !brokers) {
+        return <SkeletonText mt={5} noOfLines={13} spacing={5} skeletonHeight={4} speed={0} />
```
Comment (malinskibeniamin) [inline frontend/src/components/pages/overview/Overview.tsx:268]: 16 -> 13?
also is it intentional that here we have speed = 0

**Review by malinskibeniamin — APPROVED** [review]
Just some small questions, we are getting close!

File: frontend/src/components/pages/overview/Overview.tsx (hunk 1 above)
Comment (jvorcak) [reply frontend/src/components/pages/overview/Overview.tsx:217]: I tried to check the UI when migrating and somewhere I tried to fine-tune some of the places. The original and this component are not exactly 1:1

File: frontend/src/components/pages/overview/Overview.tsx (hunk 2 above)
Comment (jvorcak) [reply frontend/src/components/pages/overview/Overview.tsx:268]: For `Skeleton` you need to provide `isActive` or sth like that, for `SketetonText` it's the other way around, you need to disable animation with speed=0, so I think this is correct.

**Pull request merged**